mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -2772,9 +2772,7 @@ async fn connect_loop<Spawner: SpawnFuture>(
 
                 Either::Right(TcpStream::connect(&socket_addr).and_then(move |stream| {
                     // Can use `unwrap` cause `dns_name` is pre-checked.
-                    let dns = ServerName::try_from(dns_name.as_str())
-                        .map_err(|e| format!("{:?}", e))
-                        .unwrap();
+                    let dns = ServerName::try_from(dns_name.as_str()).unwrap();
```

Comment:
I have also done the same with:
https://github.com/KomodoPlatform/komodo-defi-framework/blob/eda0f425b4890e62cfcb5141e17aea719ef1d1af/mm2src/coins/utxo/rpc_clients.rs#L2756
since if it fails it should no longer succeed later.